backend/db.js

```diff
@@ -41,22 +40,16 @@ function addHooks(sequelize, restart) {
  * @param {string | Array} dbConfig.url DB connection URL or an array of URLs
  * @param {Object} dbConfig.options DB connection options
  * @param {Object} loggerFactory object containing `getLogger` function
- * @param {string} modelsDir directory for models lookup
- * @param {Array} modelExcludes list of files to be excluded from model definitions discovered by reading models
- * directory as specified by `modelsDir` argument
+ * @param {(function(Sequelize, Sequelize.DataTypes): Sequelize.Model)[]} models array of functions returning sequelize model
```

Comment:
I'm open for name suggestions for this parameter, if you think it is not accurate.

Reply:
I guess it's a factory function pattern, so maybe `modelFactories`?

Reply:
Yes, make sense. Updated.